akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -161,6 +161,13 @@ akka.cluster.sharding {
   # This dispatcher for the entity actors is defined by the user provided
   # Props, i.e. this dispatcher is not used for the entity actors.
   use-dispatcher = ""
+
+  # Set to a lease implementation to have shards acquire a lease before
+  # initializing
+  lease-implementation = ""
```

Comment:
Comment should describe that it's a config path. Maybe `lease-implementation is a bad name for this property, I'm associating implementation with fqcn. For a few dispatcher settings we have `use-dispatcher`, pointing to a dispatcher conf. Should we name it `use-lease`?

Reply:
Yep, will rename